CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 2.5.0
```

Comment:
I think it's a quite small addition, should it be this just 2.4.1 ?

Reply:
Well, it's not a fix either, so should be 2.5.0